packages/terra-notification-dialog/translations/sv-SE.json

```diff
@@ -1,6 +1,6 @@
 {
-  "Terra.notification.dialog.hazard-low": "Information",
-  "Terra.notification.dialog.error": "Fel",
-  "Terra.notification.dialog.hazard-medium": "Varning",
-  "Terra.notification.dialog.hazard-high": "Avisering"
+  "Terra.notification.dialog.hazard-low":	"Information",
```

Comment:
Is this the proper translated word?  It appears to be English.

Reply:
It was provided that way:
<img width="688" alt="Screenshot 2024-04-15 at 10 22 58 AM" src="https://github.com/cerner/terra-framework/assets/119358186/67e4438e-c878-4bd3-9b20-1e959843ecb6">

And it looks like it's the same in English and Swedish:
<img width="1303" alt="Screenshot 2024-04-15 at 10 25 11 AM" src="https://github.com/cerner/terra-framework/assets/119358186/73fdf092-ae84-49ed-a389-1d29bfc21060">